src/main/webapp/resources/js/pages/announcement/AnnouncementDashboard.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import { List, notification, Typography } from "antd";
+import { fromNow } from "../../utilities/date-utilities";
+import {
+  getUnreadAnnouncements,
+  markAnnouncementRead,
+} from "../../apis/announcements/announcements";
+import ViewUnreadAnnouncement from "./ViewUnreadAnnouncement";
+import Markdown from "react-markdown";
+import { PriorityFlag } from "./PriorityFlag";
+
+export function AnnouncementDashboard() {
+  const [unreadAnnouncements, setUnreadAnnouncements] = useState([]);
+  const [unreadTotal, setUnreadTotal] = useState(0);
+  const { Paragraph } = Typography;
+
+  useEffect(() => {
+    getUnreadAnnouncements().then((data) => {
+      setUnreadAnnouncements(data.data);
+      setUnreadTotal(data.data.length);
+    });
+  }, [unreadTotal]);
+
+  function markAnnouncementAsRead(aID) {
+    return markAnnouncementRead({ aID })
+      .then(() => {
+        setUnreadTotal(unreadTotal - 1);
+      })
+      .catch(({ message }) => {
+        notification.error({ message });
+      });
+  }
+
+  return (
+    <List
+      pagination={unreadTotal > 5 ? { pageSize: 5 } : false}
+      dataSource={unreadAnnouncements}
+      renderItem={(item) => (
+        <List.Item className="t-announcement-item">
+          <List.Item.Meta
+            avatar={<PriorityFlag hasPriority={item.priority} />}
```

Comment:
Can you update this to be an actual avatar:

```

              <Avatar
                style={{ backgroundColor: "#fff" }}
                icon={<PriorityFlag hasPriority={item.priority} />}
              />
```

Reply:
Please see c74c721758aa2f74939229fc7b6a3ff651d13ea4.